apps/charterafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,176 @@
+import link from "../fields/link";
+import richText from "../fields/richText";
+import {
+  validateHexColor,
+  blendModeOptions,
+  MuiButtonColors,
+} from "../utils/colors";
+
+const Hero = {
+  slug: "hero",
+  fields: [
+    {
+      name: "slides",
+      type: "array",
+      minRows: 1,
+      fields: [
+        {
+          name: "title",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "color",
+              type: "text",
+              validate: validateHexColor,
+              required: true,
+              label: {
+                en: "Title Color",
+                fr: "Couleur du titre",
+                pt: "Cor do título",
+              },
+            },
+            richText({
+              name: "content",
+              required: true,
+              admin: {
+                elements: ["h2", "h3", "h4", "h5", "h6", "link"],
+                leaves: ["bold", "italic", "underline", "code"],
+              },
+              localized: true,
+              label: {
+                en: "Title",
+                fr: "Titre",
+                pt: "Título",
+              },
+            }),
+          ],
+        },
+        {
+          name: "subheading",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "color",
+              type: "text",
+              validate: validateHexColor,
+              required: true,
+              label: {
+                en: "Subheading Color",
+                fr: "Couleur du sous-titre",
+                pt: "Cor do subtítulo",
+              },
+            },
```

Comment:
Again, label should be just **Color**; subheading is already on the group. Do the same for background, etc.